app/controllers/authentication_controller.rb

```diff
@@ -2,8 +2,8 @@ class AuthenticationController < ApplicationController
   skip_before_action :authorize_request, only: :authenticate
   # return auth token once actor is authenticated
   def authenticate
-    auth_token = AuthenticateActor.new(auth_params[:code]).call
-    #auth_token = JsonWebToken.encode(actor_id: "1")
+    #auth_token = AuthenticateActor.new(auth_params[:code]).call
+    auth_token = JsonWebToken.encode(actor_id: "1")
```

Comment:
I suggest adding a `Warning.warn` to have the annoying reminder in stderr that this is bypassed.

Reply:
@taliaga Added warning